modules/antismash/antismashlitedownloaddatabases/meta.yml

```diff
@@ -0,0 +1,44 @@
+name: antismash_antismashlitedownloaddatabases
+description: antiSMASH allows the rapid genome-wide identification, annotation and analysis of secondary metabolite biosynthesis gene clusters. This module downloads the antiSMASH databases.
+keywords:
+  - secondary metabolites
+  - BGC
+  - biosynthetic gene cluster
+  - genome mining
+  - NRPS
+  - RiPP
+  - antibiotics
+  - prokaryotes
+  - bacteria
+  - eukaryotes
+  - fungi
+  - antismash
+  - database
+tools:
+  - antismash:
+      description: antiSMASH - the antibiotics and Secondary Metabolite Analysis SHell
+      homepage: https://docs.antismash.secondarymetabolites.org
+      documentation: https://docs.antismash.secondarymetabolites.org
+      tool_dev_url: https://github.com/antismash/antismash
+      doi: "10.1093/nar/gkab335"
+      licence: ['AGPL v3']
+
+input:
+  - database_dir:
+      type: directory
+      description: Directory containing the user-specified path for the download of antiSMASH databases
+      pattern: "*"
+
+output:
+  - versions:
+      type: file
+      description: File containing software versions
+      pattern: "versions.yml"
+
+  - database:
+      type: directory
+      description: Downloaded clusterblast databases
+      pattern: "lib/python3.8/site-packages/antismash/databases/"
```

Comment:
is this correct? Above you were saying this location was a user defined location?